src/common/CMakeLists.txt

```diff
@@ -13,6 +13,7 @@ SET( ${PROJECT_NAME}_HDRS
 
 SET( ${PROJECT_NAME}_SRCS
   src/CaPTkGUIUtils.cpp
+  src/CaPTkutils.cpp
```

Comment:
@saimarathore PR is failing. Did you forget the file that you added here?

Reply:
this file was empty so i removed this line from CMakeLists